Portal/Pages/Spaces/Details.cshtml

```diff
@@ -42,51 +45,83 @@
       id="dropzoneForm">
 </form>
 
-<table class="table" id="assetTable">
+<form method="post" asp-page-handler="ReOrder">
+    <input type="hidden" value="@spaceId" name="spaceId">
+    
+    <div class="btn-group" role="group" aria-label="Ordering1" id="saveOrderGroup">
+        <input type="submit" class="btn btn-primary" name="save-order" id="saveOrder" value="Save Order" />
+    </div>
+    <table class="table" id="assetTable">
     <thead>
     <tr>
         <th scope="col"></th>
         <th scope="col">n1</th>
         <th scope="col">Name</th>
         <th scope="col">Created</th>
+    @if (Model.SpacePageModel.Space.IsManifestSpace())
+    {
+        <th></th>
+    }
     </tr>
     </thead>
-    <tbody>
-    @foreach (var image in Model.SpacePageModel.Images.Members)
+<tbody>
+@{
+    int rowIndex = 0;
+    foreach (var image in Model.SpacePageModel.Images.Members)
     {
         <tr>
             <td>
                 @if (image.Family == 'I')
                 {
-                    <iiif-thumb 
+                    <iiif-thumb
                         customer-url-part="@Model.Customer"
                         space-url-part="@Model.Space"
                         api-image="@image"
-                        class="iiifpreview" 
+                        class="iiifpreview"
                         settings="@Model.DlcsSettings"
-                        small="28" large="200" />
+                        small="28" large="200"/>
                 }
             </td>
             <td>@image.Number1</td>
-            <td><a asp-page="/Images/Index" 
-                   asp-route-space="@Model.Space" 
+            <td>
+                <a asp-page="/Images/Index"
+                   asp-route-space="@Model.Space"
                    asp-route-image="@image.GetLastPathElement()">@image.GetLastPathElement()</a></td>
             <td>@image.Created</td>
+        @if (Model.SpacePageModel.Space.IsManifestSpace())

```

Comment:
`IsManifestSpace` is on the viewmodel now (`Model.SpacePageModel.IsManifestSpace`) but it does the exact same thing.